.prettierignore

```diff
@@ -0,0 +1,11 @@
+# Copyright Envoy AI Gateway Authors
+# SPDX-License-Identifier: Apache-2.0
+# The full text of the Apache license is available in the LICENSE file at
+# the root of the repo.
+
+manifests/charts/ai-gateway-helm
+manifests/charts/ai-gateway-crds-helm
+tests/internal/testopenai/cassettes
+tests/internal/testopeninference/spans
+cmd/aigw/testdata
+internal/envgen/testdata
```

Comment:
I added this folder since I think it shouldn't be formatted for the same reason as aigw/testdata